documentation/source/intro/overview.rst

```diff
@@ -11,7 +11,7 @@ OpenEBS is Container Attached Storage(CAS) - which combines the best of DAS and
 
 You can try out OpenEBS on your Kubernetes Cluster using the `Quick Start Guide`_. 
  
-.. _Quick Start Guide: http://openebs.readthedocs.io/en/latest/getting_started/quick_install.html
+.. _Quick Start Guide: ./quick_install.html
```

Comment:
Quick Start Guide link is not working,give a correct path to quick_install.html file

Reply:
verified that the link works in generated documentation. http://openebs-kmova.readthedocs.io/en/update-user-doc/intro/overview.html